RELNOTES

```diff
@@ -48,6 +48,7 @@ firejail (0.9.73) baseline; urgency=low
   * legal: selinux.c: Split Copyright notice & use same license as upstream
     (#5667)
   * new profiles: fix-qdf, qpdf, zlib-flate, standard-notes, url-eater
+  * new profiles: rssguard
```

Comment:
Suggestion: The list of profiles added can be auto-generated by git (such as
near the release date), so I'd leave RELNOTES as is to avoid potential
conflicts between PRs.